src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
Combine with `background ` shorthand property

Reply:
@loralevitska  Is it like?
```
background: url(../images/star.svg) no-repeat center;
```